pkg/operator/ceph/object/rgw.go

```diff
@@ -342,6 +342,10 @@ func validateStore(context *clusterd.Context, s cephv1.CephObjectStore) error {
 	if s.Namespace == "" {
 		return errors.New("missing namespace")
 	}
+	securePort := s.Spec.Gateway.SecurePort
+	if securePort < 0 || securePort > 65535 {
```

Comment:
Does this check support the secure port being unset? I assume this is the zero case?

Reply:
That's correct; 0 is the default, unset case.